app/components/widgets/ProgressSteps.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component, Element } from 'react';
+import { observer } from 'mobx-react';
+import classNames from 'classnames';
+import SvgInline from 'react-svg-inline';
+
+import iconTickSVG from '../../assets/images/widget/tick.inline.svg';
+import iconCrossSVG from '../../assets/images/widget/cross.inline.svg';
+import styles from './ProgressSteps.scss';
+
+type Props = {
+  stepsList: Array<string>,
+  progressInfo: {
+    currentStep : number, // example, 0 = pointing to stepsList[0]
```

Comment:
maybe `currentStep` and `stepState` should be enums instead of commenting what numbers mean what in a few different places?

Reply:
`widgets/ProgressSteps.js` is widgets so basic idea is to make it reusable, 
for Trezor Connect we have steps with name `ABOUT`, `CONNECT` and `SAVE` 
suppose if we try to reuse for new functionality with steps `A`, `B` and `C`,  hence i tried to refrain from being specific.

Reply:
right now it has three state for current step, but there is not any change in 0 = LOAD or 1 = PROCESS (which is not required for current specification ) , only change in ERROR state.